client-side/src/stories/stylies.scss

```diff
@@ -0,0 +1,5 @@
+$background-class: #ffffff;
+$secondary-color-hard:  rgb(200, 245, 232);
+$primary-color: black;
+$secondary-color: rgb(103, 252, 210);
+@import url('https://fonts.googleapis.com/css2?family=Nanum+Gothic&display=swap');
```

Comment:
please move all variables to variables file, this stiles file already deleted from main branch